server/index.js

```diff
@@ -1,7 +1,10 @@
 const express = require('express');
 const path = require('path');
+const router = require('./router');
 
 const app = express();
+app.use(express.json());
+app.use('/', router);
 app.use(express.static(path.join(__dirname, '../client/dist')));
 
 const PORT = process.env.PORT || 3000;
```

Comment:
right here we reference process.env.PORT and it is the highest level in our server code so the .env file should be in here and a PORT variable added to the example.env file.